src/pages/workspaces/workspace/Dashboard/Dashboard.ts

```diff
@@ -493,16 +492,14 @@ const WorkspaceDashboardComponent = (
   ]);
 };
 
-const WorkspaceDashboard: (
-  props: WorkspaceDashboardProps
-) => ForwardRefRenderFunction<typeof WorkspaceDashboardComponent, WorkspaceDashboardProps> = _.flow(
```

Comment:
This type was incorrect. WorkspaceDashboard is a component that returns a ReactNode, not a render function.